terrasnek/workspaces.py

```diff
@@ -98,7 +98,7 @@ def list_all(self, include=None):
 
         Returns an object with two arrays of objects.
         """
-        return self._list_all(self._org_api_v2_base_url, include=include)
+        return self._list_all(self._org_api_v2_base_url, search=None, include=include)
```

Comment:
This should be `search=search`, otherwise the search will always be for a `None` value.

Reply:
/facepalm. This is now resolved.